packages/rum/run-script.js

```diff
@@ -23,93 +23,9 @@
  *
  */
 
-const path = require('path')
 const JasmineRunner = require('jasmine')
-const express = require('express')
-const serveIndex = require('serve-index')
 const testUtils = require('../../dev-utils/test-utils')
-const { runIntegrationTest } = require('./test/e2e/integration-test')
-const { generateNotice } = require('../../dev-utils/dep-info')
-
-const PROJECT_DIR = path.join(__dirname, './')
-
-function createBackendAgentServer() {
```

Comment:
@jahtalab somehow we missed to remove this while refactoring and this code was repeated twice.